apps/ledger-live-desktop/src/renderer/screens/asset/AssetBalanceSummaryHeader.tsx

```diff
@@ -122,6 +122,7 @@ export default function AssetBalanceSummaryHeader({
         defaultAccount: account,
         defaultParentAccount: parentAccount,
         defaultCurrency: currency,
+        defaultAmountFrom: "0",
```

Comment:
you type is `Number` not string

Reply:
The type itself was wrong. 
It should be a string indeed, but the name was misspelled in the type `defaultFromAmount` -> `defaultAmountFrom`.